### PR TITLE
Ibp 4746 germplasm details audit changes 2

### PR DESCRIPTION
**Pull request by lik-83**

### CODE REVIEW

File: src/main/jhipster/src/main/webapp/app/entities/germplasm/attribute/germplasm-attribute-type.ts
Comment (nahuel-soldevilla) [inline src/main/jhipster/src/main/webapp/app/entities/germplasm/attribute/germplasm-attribute-type.ts:4]: Later we can see if it's possible to use ids here and reuse eventually the same enum as the attribute-to-ontology branch: https://github.com/IntegratedBreedingPlatform/Workbench/commit/9aef8dcca67ce06d803fe208bdbb20eb79c7d4c9
But this should be ok for now, or let's wait until that one is merged and we can reuse it

File: src/main/jhipster/src/main/webapp/app/germplasm-details/audit/attributes/germplasm-attribute-audit.route.ts
Comment (nahuel-soldevilla) [inline src/main/jhipster/src/main/webapp/app/germplasm-details/audit/attributes/germplasm-attribute-audit.route.ts:6]: Not mandatory but, if you can rename it to `/attributes/`, we'll be following the same RESTful URL pattern as the API, in which we use plurals for resources (same for `/name/`)

File: src/main/jhipster/src/main/webapp/app/germplasm-details/audit/attributes/germplasm-attribute-audit.component.html
Comment (nahuel-soldevilla) [inline src/main/jhipster/src/main/webapp/app/germplasm-details/audit/attributes/germplasm-attribute-audit.component.html:31]: a new class (e.g `.diff-new { ...`) will be more appropiate

File: src/main/jhipster/src/main/webapp/app/shared/util/date-utils.ts
Comment (nahuel-soldevilla) [inline src/main/jhipster/src/main/webapp/app/shared/util/date-utils.ts:7]: Nice I'll try to cherry pick this, I need it to validate variable type dates in germp import

**Review by nahuel-soldevilla — APPROVED** [review]
It's looking great :+1: 
As you mentioned, there's a lot of potential for reusing some elements, but we can improve it later

File: src/main/jhipster/src/main/webapp/app/germplasm-details/audit/attributes/germplasm-attribute-audit.route.ts
Comment (lik-83) [reply src/main/jhipster/src/main/webapp/app/germplasm-details/audit/attributes/germplasm-attribute-audit.route.ts]: Done!

File: src/main/jhipster/src/main/webapp/app/shared/util/date-utils.ts
Comment (nahuel-soldevilla) [reply src/main/jhipster/src/main/webapp/app/shared/util/date-utils.ts:7]: https://github.com/IntegratedBreedingPlatform/Workbench/commit/8f6f6ea6a940d89c760fbdf10fde480e789e77e2

**Pull request merged**
